Jenkinsfile

```diff
@@ -16,9 +16,48 @@ stage("Unit Test") {
         export LD_LIBRARY_PATH=/usr/local/cuda-10.1/lib64
         export MPLBACKEND=Agg
         export MXNET_CUDNN_AUTOTUNE_DEFAULT=0
-        python3 -m pip install --upgrade --force-reinstall -e .
-        python3 -m pip install pytest
+
```

Comment:
we can just remove the conda env, and rebuilt it for each test. one issue here is assigning a uniqle name to each executor to allow parallel testing, you could refer to d2l's setup https://github.com/d2l-ai/d2l-en/blob/master/Jenkinsfile#L14